draw.js

```diff
@@ -875,6 +878,7 @@ voteBox: function (page) {
 						button.setAttribute('data-selected', 'true')
 						replaceVote('.voteCount[data-vote="' + state + '"]', 1)
 					}
+					button.disabled = false
```

Comment:
to check if a request succeeded:
- successful request: `e` is falsey, response in `resp`
- error: `e` is error type, *sometimes* `resp` has an error message (depending on error type)

and make sure you unlock the button even if the request failed, so the user can try voting again

Reply:
yeah um that's what i had to update, i just hacked like a dummie